src/navigators/DrawerNavigator.js

```diff
@@ -32,6 +32,7 @@ const DefaultDrawerConfig = {
     Dimensions.get('window').width - (Platform.OS === 'android' ? 56 : 64),
   contentComponent: DrawerItems,
   drawerPosition: 'left',
+  useNativeAnimations: false,
```

Comment:
One thing: I think this should be set to true

Reply:
I defaulted to false because I didn't want to accidentally break someones app. Happy to change it though.

Reply:
Yeah I don't think it would have any negative ramifications to set this to default.

Reply:
Ok done!